### PR TITLE
MNT: Fixes for new deployment

### LINKED ISSUE #112: Lightpath opens extra PyDM Main Window
## Expected Behavior

Should not spawn an empty main window

## Current Behavior

Extra main window loaded
image

## Possible Solution

Read PyDM docs and update lightpath code

## Steps to Reproduce (for bugs)

1. Open lightpath UI

## Context

If it annoys me, it will annoy users

## Your Environment

Latest

### LINKED ISSUE #114: Lightpath Ignores Happi's Lightpath Flag
## Expected Behavior

Expect `active=True`, `lightpath=True` -> use in the path

## Current Behavior

`active=True`, `lightpath=False` -> is also used in the path

## Possible Solution

extend the happi load calls

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

Accumulated small fixes PR for lightpath redeployment
closes #112 
closes #114 
partially addresses #116 probably

### CODE REVIEW

File: lightpath/ui/widgets.py [hunk 1]
```diff
@@ -120,13 +120,16 @@ class LightRow(InactiveRow):
     parent : QObject, optional
     """
     MAX_HINTS = 2
+    device_updated = pyqtSignal()
```
Comment (hhslepicka) [inline lightpath/ui/widgets.py:123]: maybe switch it to use `qtpy` and `Signal` instead of `pyqtSignal` ?

File: lightpath/ui/widgets.py [hunk 2]
```diff
@@ -187,7 +193,8 @@ def update_state(self, *args, **kwargs):
         self.state_label.setText(self.last_state.name)
         color = state_colors[self.last_state.value]
         style_color = to_stylesheet_color(color)
-        self.state_label.setStyleSheet("QLabel {color: %s}" % style_color)
+        style_sheet = "QLabel {color: %s}" % style_color
```
Comment (hhslepicka) [inline lightpath/ui/widgets.py:196]:
```suggestion
        style_sheet = f"QLabel {{color: {style_color} }}"
```

File: lightpath/ui/widgets.py (hunk 1 above)
Comment (ZLLentz) [reply lightpath/ui/widgets.py:123]: https://github.com/pcdshub/lightpath/blob/415410dfa810af825337580c437409b51b6a416d/lightpath/ui/widgets.py#L9

Comment (ZLLentz) [reply lightpath/ui/widgets.py:123]: I guess this is how Teddy updated to qtpy. Should I take the opportunity to make it clearer by removing the as import?

Comment (hhslepicka) [reply lightpath/ui/widgets.py:123]: It seems like a good idea!

File: lightpath/ui/widgets.py (hunk 2 above)
Comment (ZLLentz) [reply lightpath/ui/widgets.py:196]: I'm way too scared of the nested `}` here to put this in

**Pull request merged**